docs/application/dotnet/guides/nui/layer.md

```diff
@@ -39,12 +51,41 @@ layer.Add(_imageView);
 
 ### Re-ordering layers
 
-A range of functions are provided to change the draw order of the layers.
+The default root layer obtained from the Window instance has a **Depth** field value of 0, which is the same as for a newly created layer. You can add a layer to the window using `window.AddLayer()` API and set the  **Depth** value as shown in the following code example: 
+
+```
+Window window = Window.Instance;
+//window.GetDefaultLayer().Depth = 0;
+
+Layer ly0 = new Layer();
+//ly0.Depth = 0
+window.AddLayer(ly0);
+//ly0.Depth = 1;
+
+Layer ly1 = new Layer();
+//ly1.Depth = 0;
+window.AddLayer(ly1);
+//ly1.Depth = 2;
+```
+
+The renderer draws layers and its content from the layers with the lowest depth at the beginning. As a result, a layer and its content with a higher depth value are drawn at the top of the stack.
+
+To reorder layers, the following API set can be used: 
+ - `Lower()`: decrements the **Depth** parameter 
+ - `LowerToBottom()`: sets the **Depth** parameter to 0. It reorders other layers, and increments the root layer **Depth**
+ - `MoveAbove(Layer l)`: moves the layer **Depth** directly above the giver layer
```

Comment:
@mihashco Just a doubt, should it be **given layer** instead of **giver layer** 
If it is **giver layer** only, ignore this comment. 
 - `MoveAbove(Layer l)`: moves the layer **Depth** directly above the given layer

Reply:
It is a typo. Sorry for that.